yarn.lock

```diff
@@ -21,447 +21,351 @@
     "@jridgewell/trace-mapping" "^0.3.24"
 
 "@apideck/better-ajv-errors@^0.3.1":
-  version "0.3.3"
```

Comment:
Where do all the changes come from? Is this related to the postcss update? It also looks like, that it includes other changes. Can you try to only include the necessary changes for updating postcss and nth-check?

Reply:
comes from indirectly updating postcss by actually updating resolve-url-loader

Reply:
When I apply the same resolution, it changes only a few dependencies. Please re-check

Reply:
mondays...